test/acceptance/full-dump-replication-test.js

```diff
@@ -96,49 +96,54 @@ describe('Acceptance | steps | fullReplicationAndEnrichment', () => {
     });
   });
 
-  describe('should import Airtable data', () => {
-    before(() => steps.importAirtableData(configuration));
+  describe('should import learning content', () => {
+    before(async function() {
+      const fullLearningContent = mockLcmsGetAirtable();
+      sinon.stub(lcms, 'getLearningContent').resolves(fullLearningContent);
+
+      await steps.importLearningContent(configuration);
+    });
 
     it('should import areas ', async () => {
       // then
       const result = await getCountFromTable({ targetDatabase, tableName: 'areas' });
-      expect(result).to.be.above(0);
+      expect(result).to.equal(2);
```

Comment:
Les enregistrements réels sont un peu cachés dans l'histoire et l'assertion n'est pas claire. 
Je propose 
```suggestion
      expect(result).to.equal(fullLearningContent.areas.length);
```